src/logic/voice/voice.ts

```diff
@@ -120,7 +153,11 @@ export function createVoiceComponent(
     }
 
     if (disconnectReason === DisconnectReason.ROOM_DELETED) {
+      // Room was already deleted by LiveKit, get participant count before cleaning up DB
+      const participantCount = await voiceDB.getCommunityVoiceChatParticipantCount(roomName)
       await voiceDB.deleteCommunityVoiceChat(roomName)
+      // Publish event after cleanup
+      await publishCommunityStreamingEndedEvent(roomName, participantCount)
```

Comment:
The `handleCommunityParticipantLeft` function be executed for all participants that got disconnected from a room. In this case, the deletion of the room could trigger an event for all users. We might want to only send an event when the room is deleted. WDYT?